supernode/store/store_mgr.go

```diff
@@ -38,21 +40,52 @@ func Register(name string, builder StorageBuilder) {
 
 // Manager manage stores.
 type Manager struct {
+	cfg *config.Config
```

Comment:
HomeDir is enough instead of the entire config. WDYT?

Reply:
`HomeDir` is enough for this storage manager, but there will be several different managers required different fields of `Config` but have similar behaviors, we may extract an abstract common implementation from them. And IMOP, the initialized `config.Config` instant should be visible for these managers.